### PR TITLE
Add Bspline

### PR DESCRIPTION
**Pull request by AuguB**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by AuguB** [comment]
Hi @conda-forge/staged-recipes, this is ready for review, thanks!

**Comment by AuguB** [comment]
@conda-forge/help-python

File: recipes/bspline/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```
Comment (synapticarbors) [inline recipes/bspline/LICENSE.md:1]: This file is already included in the source distribution, so you don't need to include it again here.

Comment (AuguB) [reply recipes/bspline/LICENSE.md]: HI, thanks for reviewing this. I just removed the LICENSE.md file, and the license_file entry in meta.yaml, but now the linter complains that that entry is needed. If I put the entry back without the file itself, the builder throws an error. What's the best way to do this?

**Pull request merged**